doc/web_servers.rst

```diff
@@ -57,7 +57,9 @@ The **minimum configuration** to get your application running under Nginx is:
         # location ~ ^/(index|index_dev)\.php(/|$) {
         location ~ ^/index\.php(/|$) {
             # the ubuntu default
-            fastcgi_pass   unix:/var/run/php5-fpm.sock;
+            fastcgi_pass   unix:/var/run/php/php7.0-fpm.sock;
+            # for running php5
+            # fastcgi_pass   unix:/var/run/php5-fpm.sock;
```

Comment:
between 5 and 7 the default directory to run the `sock` on changed for Ubuntu from:
`/var/run/` to `var/run/php`
minor diff but still very useful to know imho